test/unit/spread.spec.ts

```diff
@@ -76,6 +76,7 @@ describe("in function call", () => {
         {
             [tstl.LuaTarget.Universal]: builder => builder.tap(expectLualibUnpack).expectToMatchJsResult(),
             [tstl.LuaTarget.LuaJIT]: builder => builder.tap(expectUnpack),
+            [tstl.LuaTarget.Lua50]: builder => builder.tap(expectUnpack).expectToMatchJsResult(),
```

Comment:
You cannot expect to match JS result here (as the test failures show) because we cannot execute Lua 5.0 yet

Reply:
Since @lolleko is working on bindings for 5.0, I'll keep the match JS result here.